docker-compose.yml

```diff
@@ -35,7 +35,11 @@ services:
       - default
 
   app:
-    build: .docker/php8.0-apache/
+    build:
+      context: .docker/php8.0-apache/
+      args:
+        user: king
+        uid: 1000
```

Comment:
Do know that this only works for systems that have 1000 as the first uid (like ubuntu)

Reply:
Good to know, but since it works in `php8.0-apache`, guess we'll see if it keeps working on other (newer) images ;)